packages/remark-custom-blocks/src/index.js

```diff
@@ -1,7 +1,7 @@
 const spaceSeparated = require('space-separated-tokens')
 
 function escapeRegExp (str) {
-  return str.replace(/[-[]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&')
+  return str.replace(new RegExp(`[-[\\]{}()*+?.\\\\^$|/]`, 'g'))
```

Comment:
This would replace any match with `undefined`:

```js
> escapeRegExp('[AA-AA|AA]AA')
'undefinedAAundefinedAAundefinedAAundefinedAA'
```

Reply:
I've updated the replace string in a commit, I had forgotten it.